lib/experimental/fonts-library/class-wp-fonts-library.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
Add the file's opening docblock, similar to the other class files in this PR.

Add a `class_exists()` guard to avoid loading this class into memory after it is introduced into WordPress Core. See comment here https://github.com/WordPress/gutenberg/pull/52704/files#r1269629926.

Reply:
Done!